lib/generators/components/templates/component.yml.erb

```diff
@@ -0,0 +1,11 @@
+name: <%= @public_name %>
+description: add description here
+body: |
+  add body here
+accessibility_criteria: |
+  The component must:
+
+  - add accessibility criteria here
+  - add accessibility criteria here
+fixtures:
+  default: {}
```

Comment:
With the exception of the name and the default fixture, where we've used placeholder values, could it match this: https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#example-yaml-file